.github/ISSUE_TEMPLATE/release.md

```diff
@@ -89,7 +89,7 @@ git push upstream v0.6.0
 28.  - [ ] Aggregate all the changelog notices from the PRs and add them to the changelog file
 29.  - [ ] Bump the versions on master from `0.9.0-SNAPSHOT` to `0.10.0-SNAPSHOT`, `0.6.0.dev0` to `0.7.0.dev0`
 30.  - [ ] Apply changes to the minor_release and patch_release issue templates if they need any
```

Comment:
Agree with your comment above. Let's fix this line.

Reply:
fixed